numpy/core/getlimits.py

```diff
@@ -156,8 +156,8 @@ def _register_known_types():
 
     # Known parameters for IEEE 754 128-bit binary float
     ld = ntypes.longdouble
-    epsneg_f128 = exp2(ld(-113))
-    tiny_f128 = exp2(ld(-16382))
+    epsneg_f128 = 2 ** -113.0
```

Comment:
@eric-wieser can I actually do this? I also noticed we have `exp2l` buried somewhere--regardless `getlimits.py` is having issues with these exp2 calls involving long double.

Reply:
No, this will underflow and produce 0 on all platforms if you don't cast an operand to longdouble

Reply:
Reverted -- no longer causes issues, thanks.